src/cmd/kssh/kssh.go

```diff
@@ -52,12 +52,30 @@ func doAction(action Action, keyPath string, remainingArgs []string) {
 	if action == SSH {
 		runSSHWithKey(keyPath, remainingArgs)
 	} else if action == Provision {
-		err := kssh.AddKeyToSSHAgent(keyPath)
-		if err != nil {
-			fmt.Printf("%v\n", err)
-			os.Exit(1)
-		}
-		fmt.Printf("Provisioned new SSH key at %s\n", keyPath)
+		provision(keyPath)
+	}
+}
+
+func provision(keyPath string) {
+	err := kssh.AddKeyToSSHAgent(keyPath)
+	if err != nil {
+		fmt.Printf("%v\n", err)
+		os.Exit(1)
+	}
+	user, err := kssh.GetDefaultSSHUser()
+	if err != nil {
+		fmt.Printf("Failed to retrieve default SSH user: %v\n", err)
+		os.Exit(1)
+	}
+	err = kssh.CreateDefaultUserConfigFile(keyPath)
+	if err != nil {
+		fmt.Printf("Failed to create ssh config file default user: %v\n", err)
+		os.Exit(1)
+	}
+	fmt.Printf("Provisioned new SSH key at %s\n", keyPath)
+	if user != "" {
+		fmt.Println("See docs/troubleshooting.md for information on configuring scp, rsync, etc to " +
+			"use the configured kssh default user")
```

Comment:
```suggestion
			"use the provisioned kssh default user")
```
??

Reply:
it's going to show this message every time? i can see that being a little annoying. maybe this should be in verbose logging?

Reply:
I think "configured" is probably the better word here since this data is in ~/.ssh/kssh-config.json. 

But yes, it would show every time. Which isn't great but I'm not sure it makes sense to only show that message with verbose logging. Because I think it is a better user experience if we tell them about those docs immediately rather than making them search for it. And the line is only printed if there is a configured default ssh user (which most people won't have). Do you have any idea on how to be less annoying while still making sure this info is front and center?